proto/substrait/algebra.proto

```diff
@@ -88,7 +88,8 @@ message ReadRel {
 
   // A table composed of literals.
   message VirtualTable {
-    repeated Expression.Literal.Struct values = 1;
+    repeated Expression.Literal.Struct values = 1 [deprecated = true];
+    repeated Expression expression_values = 2;
```

Comment:
minor/nit: I'm partial to `expressions` instead of `expression_values` as it's less verbose an tacking `_values` on the end feels redundant. I'm not wedded to this.

Reply:
fixed it